ironblogger/static/style.css

```diff
@@ -81,6 +81,12 @@ section {
   overflow:hidden;
 }
 
+section > p > a {
```

Comment:
Ah! `display: block` plus `margin: 0 auto` does work! I hadn't tried that particular combo. Shall i backtrack and open a new PR?